src/components/history/GitOpsDeploymentHistory.tsx

```diff
@@ -0,0 +1,120 @@
+import './GitOpsDeploymentHistory.scss';
+
+import * as _ from 'lodash';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+// import { RouteComponentProps } from 'react-router-dom';
+
+import {
+  ListPageFilter,
+  RowFilter,
+  useListPageFilter,
+  VirtualizedTable,
+} from '@openshift-console/dynamic-plugin-sdk';
+import { LoadingBox } from '@patternfly/quickstarts';
+
+import GitOpsEmptyState from '../GitOpsEmptyState';
+import { GitOpsHistoryData } from '../utils/gitops-types';
+import { getEnvData } from '../utils/gitops-utils';
+import { GitOpsDeploymentHistoryColumns } from './GitOpsDeploymentHistoryColumns';
+import { GitOpsDeploymentHistoryTableRow } from './GitOpsDeploymentHistoryTableRow';
+
+type GitOpsDeploymentHistoryProps = {
+  customData: {
+    emptyStateMsg: string;
+    envs: string[];
+    applicationBaseURI: string;
+  };
+};
+const GitOpsDeploymentHistory: React.FC<GitOpsDeploymentHistoryProps> = ({
+  customData: { emptyStateMsg, envs, applicationBaseURI },
+}) => {
+  const { t } = useTranslation();
+  const columns = GitOpsDeploymentHistoryColumns();
+  const envRowFilters: RowFilter[] = [
+    {
+      filterGroupName: t('gitops-plugin~Environment'),
+      type: 'environment',
+      reducer: (s: GitOpsHistoryData): string => s?.environment,
```

Comment:
Something is not quite right about the filter.  If you enable all the environments in the filter, all the rows should still remain.  Instead, it shows nothing is found.  I think it should be an OR operation, and not an AND operation when multiple filters are added

Reply:
Thanks for the catch.

Reply:
Looks good